api/v1beta1/hosted_controlplane.go

```diff
@@ -211,16 +219,15 @@ type HostedControlPlaneStatus struct {
 	// +kubebuilder:validation:Optional
 	OAuthCallbackURLTemplate string `json:"oauthCallbackURLTemplate,omitempty"`
 
-	// Version is the semantic version of the release applied by
-	// the hosted control plane operator
-	// +kubebuilder:validation:Optional
-	Version string `json:"version,omitempty"`
-
-	// ReleaseImage is the release image applied to the hosted control plane.
-	ReleaseImage string `json:"releaseImage,omitempty"`
+	// Version is the status of the release version applied by the
+	// hosted control plane operator.
+	// +optional
+	Version *ClusterVersionStatus `json:"version,omitempty"`
 
 	// lastReleaseImageTransitionTime is the time of the last update to the current
 	// releaseImage property.
+	//
+	// Deprecated: Use version.history[0].startedTime instead.
```

Comment:
Same as the above comment

Reply:
This is fine as long as we don't remove it